src/components/Main.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { useBlogs } from '../hooks/useBlogs.js';
+import BlogCard from './BlogCard/BlogCard.js';
+
+export default function Main() {
+  const blogs = useBlogs();
+
+  return (
+    <main>
+      {blogs.map((blog) => (
+        <BlogCard key={blog.id} title={blog.title} subtitle={blog.subtitle} text={blog.text} image={blog.image} />
```

Comment:
If you are going to pass every property of an object as a prop to your component you can take advantage of the spread operator to make less work for yourself. Consider changing this to the following: 

```js
<BlogCard key={blog.id} {...blog} />